templates/docs/base/forms.md

```diff
@@ -64,6 +64,16 @@ To use checkbox component in headings use the `.p-checkbox--heading` variant of
 View example of the heading checkbox components
 </a></div>
 
+#### Tri-state checkbox
```

Comment:
As it seems to be officially called "indeterminate" or "mixed-state" in the docs, let's use that, as it better describes what this is meant to be used for.